pkg/api/handlers/libpod/play.go

```diff
@@ -70,6 +70,16 @@ func PlayKube(w http.ResponseWriter, r *http.Request) {
 		password = authConf.Password
 	}
 
+	logDriver := query.LogDriver
+	if logDriver == "" {
+		config, err := runtime.GetConfig()
+		if err != nil {
+			utils.Error(w, http.StatusInternalServerError, err)
+			return
+		}
+		query.LogDriver = config.Containers.LogDriver
```

Comment:
Nit (didn't realize in the previous review):
In case you need to repush.  It would be cleaner to not overwrite `query.LogDriver` but to use `logDriver` - it has also been declared outside the if branch indicating that it's lifespan goes beyond this branch.